Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -378,7 +380,7 @@ def generate_md_context_get_issue(data, customfields=None, nofields=None):
         custom_fields = [i for i in demisto.get(element, "fields") if "custom" in i]
         if custom_fields and customfields and not nofields:
             field_mappings = get_custom_field_names()
-            for field_returned in custom_fields:
+            for field_returned in customfields:
```

Comment:
Can you explain the rationale for this change?

Reply:
It's a bug in a contribution. When giving custom fields, it ran and returned *All field* of the issue (which because of how Jira works is all Jira fields) which contained the word "custom" instead of the custom fields we actually asked for.
Consulted with @ilaner who knows the PR for it and we agreed.

Reply:
Then please remove `custom_fields`. @yuvalbenshalom fyi.